AmericanMedicalAssociation/ama-style-guide-2
# EWL-7931: Add upvote functionality to Article pages

## Ticket(s)

**Github Issue**
N/A

**Jira Ticket**
- [EWL-7931: Add upvote functionality to Article pages](https://issues.ama-assn.org/browse/EWL-7931)
- [EWL-8077: Article Pages Upvote - Content Matrices](https://issues.ama-assn.org/browse/EWL-8077)

## Description
Adding styles and icons for Article Upvote functionality.


## To Test
- Pull branch
- Pull associated D8 PR and configure env to use local SG
- Run `gulp`
- See D8 PR for further testing instructions

## Visual Regressions
N/A


## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
